src/main/java/com/nis/Client/service/CollegeService.java

```diff
@@ -6,34 +6,4 @@
 import com.nis.Client.repository.CollegeRepository;
 @Service
 public class CollegeService {
-@Autowired
```

Comment:
Why did we remove this?

Reply:
This is not implementing interface. Hence, removed it